### PR TITLE
inject log with labels

### PR DESCRIPTION
**Pull request by wulianglongrd**
follow: https://github.com/istio/istio/pull/53106

### CODE REVIEW

File: pkg/kube/inject/webhook.go [hunk 1]
```diff
@@ -1110,13 +1111,15 @@ func (wh *Webhook) inject(ar *kube.AdmissionReview, path string) *kube.Admission
 	if pod.ObjectMeta.Namespace == "" {
 		pod.ObjectMeta.Namespace = req.Namespace
 	}
-	log.Infof("Sidecar injection request for %v/%v", req.Namespace, podName)
+
+	log = log.WithLabels("ns", req.Namespace, "pod", pod.Namespace+"/"+podName)
```
Comment (howardjohn) [inline pkg/kube/inject/webhook.go:1115]: why do we need `ns`? its already part of the `pod` label?

File: pkg/kube/inject/webhook.go [hunk 2]
```diff
@@ -1110,13 +1111,15 @@ func (wh *Webhook) inject(ar *kube.AdmissionReview, path string) *kube.Admission
...
+	log.Infof("Process sidecar injection request")
```
Comment (howardjohn) [inline pkg/kube/inject/webhook.go:1116]: We don't need this info logs I think, will be spammy?

File: pkg/kube/inject/webhook.go (hunk 1 above)
Comment (wulianglongrd) [reply pkg/kube/inject/webhook.go:1115]: Could they be different? See L1112

File: pkg/kube/inject/webhook.go (hunk 2 above)
Comment (wulianglongrd) [reply pkg/kube/inject/webhook.go:1116]: This line is modified based on L1113 on the left (see left, the old code), the old:
```
log.Infof("Sidecar injection request for %v/%v", req.Namespace, podName)
```
I think it should be fine to keep it, but no strong demand.

Comment (howardjohn) [reply pkg/kube/inject/webhook.go:1116]: ah my bad thanks. Let's keep it

File: pkg/kube/inject/webhook.go (hunk 1 above)
Comment (wulianglongrd) [reply pkg/kube/inject/webhook.go]: Thanks, removed `ns`.

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**